public/locales/en-US/grafana_old.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Do we need to commit these? Wasn't this added to the `gitignore` in a different branch?